doc/thingpedia-tutorial-cat.md

```diff
@@ -98,17 +98,19 @@ dataset @<your-name>.thecatapi {
 ```
 
 ## Step 4: submitting the device
-Click the `SAVE` button at the top left corner to submit the device. 
+Click the `Create` button at the top left corner to submit the device. 
 Congratulation! You made yourself a cat device in Thingpedia. 
-Go to [Thingpedia page](/thingpedia) and search for "my cat api" to see your device.
+Go to [Thingpedia page](/thingpedia) and search for your device name!
 
 ## Try your device
 Please give Almond around 5 minutes to digest the device you just created.
 A banner will tell you the status on the top of the page. 
 Once the banner disappears,
-you can go to [My Almond](/me) and type in `get a cat picture` to try out your device.
+you can go to [My Almond](/me), click on [Configure new skill](/me/devices/create) under "Enabled Skills" and then click on your device's name to add it.
+
+Then type in `get a cat picture` to get some cat pictures!
```

Comment:
You shouldn't need to enable the skill, it should come automatically the first time you issue a command?

Reply:
Not really. The issue is that because the utterances in dataset.tt is the same as every other Cat API device. Another Cat API device comes up instead of the one that I built

Reply:
As a non-admin developer (that is, anybody else than you) you cannot see the other devices, and Almond will silently ignore them. Also, it should prefer your version over the default version, because it's newer. Enabling a device has no effect on how the program is chosen among the possible candidates.

Reply:
Rightt okay I'll remove the enabling step then